### PR TITLE
Vile Rattus Cult

### PR DESCRIPTION
**Pull request by hhaddow**
Just need someone to proofread it, everything seems to be there

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ table!
 |Soul-Snatcher Cults|v2.8|Done|
 |TAO Coalition|v2.11|Done|
 |Titan Lords|v2.4|Done|
+|Vile Rattus Cult|v2.5|Awaiting Review|
```
Comment (sprutton1) [inline readme.md:96]: Can you change this to "Done" instead as this will be what ends up in the repo once the change is merged?

**Review by sprutton1 — CHANGES_REQUESTED** [review]
Instead of the .catz file, can you submit the .cat file? The catz is a zip, so I can't review the text without pulling it down.

**Comment by hhaddow** [comment]
I've swapped it out for a catz file

File: Vile Rattus Cult.cat [hunk 2]
```diff
@@ -0,0 +1,2832 @@
...
+      </profiles>
+      <infoLinks>
+        <infoLink id="5c93-7dd8-d317-173d" name="AP(X)" hidden="false" targetId="f84f-fda5-e478-455d" type="rule"/>
+        <infoLink id="7ac3-7c4e-45a3-5ee9" name="Deadly(X)" hidden="false" targetId="377b-3864-960e-57ac" type="rule"/>
+      </infoLinks>
+      <costs>
+        <cost name="pts" typeId="567f-6468-66c6-2ea2" value="0.0"/>
+      </costs>
+    </selectionEntry>
+    <selectionEntry id="08a5-2fc9-18b4-e30a" name="Tripple Pistols" hidden="false" collective="false" import="true" type="upgrade">
```
Comment (sprutton1) [inline Vile Rattus Cult.cat:1162]: Typo

File: Vile Rattus Cult.cat [hunk 3]
```diff
@@ -0,0 +1,2832 @@
...
+      <infoLinks>
+        <infoLink id="377a-5352-7057-d915" name="Blast(X)" hidden="false" targetId="187f-6414-7037-a542" type="rule"/>
+      </infoLinks>
+      <costs>
+        <cost name="pts" typeId="567f-6468-66c6-2ea2" value="0.0"/>
+      </costs>
+    </selectionEntry>
+    <selectionEntry id="ec69-63a6-384d-a390" name="Artillery Gun" hidden="false" collective="false" import="true" type="upgrade">
+      <profiles>
+        <profile id="e3f8-bb3d-c04a-b55d" name="Artillary Gun" hidden="false" typeId="a8fa-e9ce-c38a-c73e" typeName="Unit">
```
Comment (sprutton1) [inline Vile Rattus Cult.cat:1858]: *Artillery

**Review by sprutton1 — APPROVED** [review]

**Review by sprutton1 — COMMENTED** [review]
@Darguth I'm not sure if you want to pull this down and test out the file in BS. If not, I can merge it

**Comment by Darguth** [comment]
@sprutton1 I didn't comb through it with a fine-tooth brush, but general review looks good. I don't have any problems merging this.

@hhaddow Welcome, glad to see another contributor!! A few minor notes for future work/iterations:
1. Rat Daemon was classified as a Monster: Heavy but I'd normally still put this under the Hero category because it has the special rule. I did this for things like Greater Daemons etc. in other lists.
2. Noticed you didn't include options for combined units. This can make the config a bit messier, but it's a feature that was requested often. I don't think it's make-or-break, but a nice-to-have.
3. When adding army-specific special rules to, I usually will include a reference link to the document. Just looks nicer on the final army list printout and lets people know where to lookup a rule if they have the OPR printouts handy. Not critical, just a mention.

Overall though it looks great. I like the grouping you did for upgrades to map more cleanly to the OPR lists, might lift that and borrow it for future work as well.

**Comment by hhaddow** [comment]
@Darguth thanks, this was my first time working with the data side of Battlescribe so I'm glad that it made the grade, I might as well start on correcting some of the issues listed, also as Tunnel Leader is also a hero (on bike) should I give him the Hero class too

And feel free to lift the way I did the upgrades

**Comment by hhaddow** [comment]
Found a couple of errors I missed, Upgrades F, Jetpacks is 15pts, should be 20pts, rat swarms had not been assigned their swarm attacks weapon, I'm just in the process of adding options for combined units.

**Pull request merged**